admin/controllers/check-window.js

```diff
@@ -99,6 +107,35 @@ const controller = {
     }
     req.flash('info', `${checkWindow['name']} has been successfully removed`)
     return res.redirect('/check-window/manage-check-windows')
+  },
+
+  /**
+   * Edit check window form.
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+
+  getCheckWindowEditForm: async (req, res, next) => {
+    req.breadcrumbs('Manage check windows', '/check-window/manage-check-windows')
+    res.locals.pageTitle = 'Edit check window'
+    req.breadcrumbs(res.locals.pageTitle)
+    let checkWindowData
+    let checkWindowViewData
+    try {
+      checkWindowData = await checkWindowV2Service.getCheckWindow(req.params.checkWindowUrlSlug)
+      checkWindowViewData = checkWindowHelper.getEditViewData(checkWindowData)
```

Comment:
It's not all that clear to me from the code that:

1) `checkWindowHelper` is a Presenter and that 
2) it is returning a view model

@danwilliams @GuyHarwood - thoughts?

Reply:
@GeorgeChatzigiannis is this helper function preparing the data model to be shown in the view?  if so, call it a presenter.

Reply:
@GuyHarwood @jon-shipley this is updated based on feedback